contrib/snapshot/template.py

```diff
@@ -0,0 +1,43 @@
+pkgname = "snapshot"
+pkgver = "46.3"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "appstream",
+    "cargo",
+    "desktop-file-utils",
+    "gettext",
+    "gtk-update-icon-cache",
+    "meson",
+    "pkgconf",
+]
+makedepends = [
+    "gstreamer-devel",
+    "gst-plugins-bad-devel",
+    "gst-plugins-base-devel",
+    "libadwaita-devel",
+]
+depends = [
+    "gstreamer-libcamera",
+    "gstreamer-pipewire",
+    "libcamera",
+    "pipewire",
+    "wireplumber",
```

Comment:
could remove `libcamera` and `pipewire` here because they get pulled in by the other stuff

Reply:
yeah it's fine if they crom from gst

i think you don't need wireplumber either

Reply:
i added wireplumber because it was mentioned as a dep but sure

Reply:
it's a dep of anything that plays audio too, yet it's not added there either

same goes for like.. a font, an icon theme, ... base deps are not useful to add